### PR TITLE
[Create timepoint] testplan update

### PR DESCRIPTION
**Pull request by ridz1208**
## Brief summary of changes
 - clearer enumeration of possible test cases after selecting subproject
 - added project test cases

### CODE REVIEW

File: modules/create_timepoint/test/TestPlan.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Create Timepoint Test Plan:
+
+1. Access a candidate that does not have all timepoints from project created
+from Access Profile and click "Create time point"
+  [Automation Testing]
+2. Ensure that clicking on all elements of the breadcrums takes you to the
```
Comment (johnsaigle) [inline modules/create_timepoint/test/TestPlan.md]:
```suggestion
2. Ensure that clicking on all elements of the breadcrumbs takes you to the 
```

File: modules/create_timepoint/test/TestPlan.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+2. Ensure that clicking on all elements of the breadcrums takes you to the 
+appropriate page.
+  [Manual Testing]
+3. Ensure that the DCCID field displays the correct candidate's identifier.
+  [Manual Testing]
+4. Choose a subproject from dropdown and ensure that the page reloads with 
+subproject selected and also a list of visitlabels if <labelSet> is set in the config.xml.
+  [Manual Testing]
+   - Ensure that if the `config.xml` file contains a `<visitLabel subprojectID="X">` 
+   matching the subproject ID selected, a list of visits ()defined in the same file
```
Comment (johnsaigle) [inline modules/create_timepoint/test/TestPlan.md]:
```suggestion
   matching the subproject ID selected, a list of visits defined in the same file 
```

Not sure why the brackets are there? Let me know if they were intentional for whatever reason

**Comment by johnsaigle** [comment]
Small changes, otherwise looks good

File: modules/create_timepoint/test/TestPlan.md (hunk 2 above)
Comment (ridz1208) [reply modules/create_timepoint/test/TestPlan.md]: probably when i was banging my head on the keyboard from that other PR

**Pull request merged**